### ISSUE TITLE
Creating a window after a modal popup causes an assert (1.86 release)

### ISSUE THREAD

**Issue by ET3D-ET3D** [ocornut/imgui#4920]
Here's a simple example:

```
		ImGui::OpenPopup("Popup");
		if (ImGui::BeginPopupModal("Popup", NULL, ImGuiWindowFlags_AlwaysAutoResize))
		{
			ImGui::Text("In popup");
			ImGui::EndPopup();
		}

		ImGui::Begin("Window");
		ImGui::Text("Test");
		ImGui::End();
```

This asserts in the Begin function of the window on "IM_ASSERT(blocking_modal != NULL);".

I fixed it locally by replacing the assert with

```
		if (blocking_modal == NULL)
			blocking_modal = modal;
```

I don't understand the code deeply enough to know if this is the best fix, but it seems to work well.

**Comment by ocornut** [comment]
Thank you @ET3D-ET3D for reporting.
@rokups looked at it and pushed a fix 19471da

**Comment by ET3D-ET3D** [comment]
Impressively quick response. Thanks.
